### PR TITLE
CV - Set baseline for load testing

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md [hunk 1]
```diff
@@ -246,3 +253,28 @@ The following screenshots show successful and failed verifications in a deployme
 ![Failed verification step](./static/cv-sumologic-pipeline-fail.png)
 
 
+
+## Set successful verification as a baseline for load testing
+
+To set a verification as baseline for future load testing:
+
+1. Go to the Pipelines page and find the pipeline where you want to set the baseline.
```
Comment (michaelcretzman) [inline docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:261]:
```suggestion
1. In Harness, go to **Deployments**, select **Pipelines**, and find the pipeline you want to use as the baseline.
```

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md [hunk 2]
```diff
@@ -246,3 +253,28 @@ The following screenshots show successful and failed verifications in a deployme
...
+   
+2. Select the successful verification that you want to use as the baseline.
```
Comment (michaelcretzman) [inline docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:263]:
```suggestion
2. Select the successful pipeline execution with the verification that you want to use as the baseline.
```

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md [hunk 3]
```diff
@@ -246,3 +253,28 @@ The following screenshots show successful and failed verifications in a deployme
...
+   The pipeline details page is displayed.
```
Comment (michaelcretzman) [inline docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:264]:
```suggestion
   The pipeline execution is displayed.
```

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md [hunk 4]
```diff
@@ -246,3 +253,28 @@ The following screenshots show successful and failed verifications in a deployme
...
+   
+3. On the pipeline details page, navigate to the **Verify** section, and then select **Pin baseline**.
```
Comment (michaelcretzman) [inline docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:266]:
```suggestion
3. On the pipeline execution, navigate to the **Verify** section, and then select **Pin baseline**.
```

**Review by michaelcretzman — APPROVED** [review]
Approved with suggestions

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md (hunk 1 above)
Comment (SudheendraKatte) [reply docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:261]: Updated

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md (hunk 2 above)
Comment (SudheendraKatte) [reply docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:263]: Yes, you are correct.

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md (hunk 3 above)
Comment (SudheendraKatte) [reply docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:264]: Updated

File: docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md (hunk 4 above)
Comment (SudheendraKatte) [reply docs/continuous-delivery/verify/configure-cv/verify-deployments-with-elastic-search.md:266]: Updated

**Comment by SudheendraKatte** [comment]
@ravilach , please review and approve.

**Comment by SudheendraKatte** [comment]
@ravilach , thanks for the comments. Please review the updated draft and approve.

**Review by ravilach — APPROVED** [review]

**Pull request merged**